n_request.c

```diff
@@ -587,20 +590,19 @@ void NoteErrorClean(char *begin)
 }
 
 #ifndef NOTE_LOWMEM
-/**************************************************************************/
-/*!
-  @brief  n_atoh - Convert a string to hex
-  @param   p
-  The character data to convert
-  @param   maxlen
-  The maximum length of that character data
-  @returns The number as converted from hex
-*/
-/**************************************************************************/
-uint64_t n_atoh(char *p, int maxlen)
+
+/**
+ * @brief Convert a hex string to a 64-bit unsigned integer.
+ *
+ * @param p The hex string to convert.
+ * @param len The length of the hex string.
+ *
+ * @returns The converted number.
+ */
+uint64_t n_atoh(char *p, int len)
 {
     uint64_t n = 0;
-    char *ep = p+maxlen;
+    char *ep = p+len;
```

Comment:
Not sure why this was called maxlen before. It's just the length of the passed in buffer, as far as I can tell. Max seems superfluous.

Reply:
I would guess it has to do with lines numbers 612, 613, and 614.